reactive-banana/src/Reactive/Banana/Prim/Graph.hs

```diff
@@ -30,8 +30,8 @@ emptyGraph = Graph Map.empty Map.empty Set.empty
 -- | Insert an edge from the first node to the second node into the graph.
 insertEdge :: (Eq a, Hashable a) => (a,a) -> Graph a -> Graph a
 insertEdge (x,y) gr = gr
-    { children = Map.insertWith (flip (++)) x [y] (children gr)
-    , parents  = Map.insertWith (flip (++)) y [x] (parents  gr)
+    { children = Map.insertWith (++) x [y] (children gr)
+    , parents  = Map.insertWith (++) y [x] (parents  gr)
```

Comment:
To avoid confusion in the future, I suggest naming the arguments:

     Map.insertWith (\new old -> new ++ old) x [y]

Reply:
👍